README.md

```diff
@@ -9,7 +9,7 @@ You can install the app from the [Enonic Market](https://market.enonic.com), add
 
 ### Build it yourself
 
-If you wish to build it yourself, you can run the `make package` command in the main directory to build and package the application. This will create a JAR file in the `/deploy` folder which you can use to install the app.
+If you wish to build it yourself, you can run the `make install_new_dependencies` command in the main directory to install the necessary packages. Then you can run the `make package` command to build and package the application. This will create a JAR file in the `/deploy` folder which you can use to install the app.
```

Comment:
Have you tried cleaning out all the stamps and node_module folders and just running `make package` ? It has a bunch of chaining task dependencies that should automatically lead to installing all packages in front and back end before building.

Reply:
Hmm.. No I did not try that. All I did was clone the repo and try to build it. That failed until I ran make install_new_dependencies. I can try to remove stamps and node_modules, but I'm not sure how that will make a difference, since the stamps and node_modules are ignored by git and were not present when I first tried to build the project 🤔

Reply:
I meant to remove those folders to get back to the state you'd have after initial cloning and and then running make package to check if would work. :) It builds and packages fine like that on my system at least, so I'm not sure why it wouldn't on yours, unless it's a Mac thing? 🤔

Reply:
Ah because of the package upgrades? Yeah, I can test that.

I would not be surprised if it is a mac thing tbh... 🤷‍♀️

Reply:
Jeg fikk bygd med `make package` da jeg klonet. Prøvde 2 ganger: en fra `main` og en med `feature` branch. Det tok 10 min første gang jeg bygget så vet ikke helt hva som skjer, men rart er det 🤷

Reply:
Jeg ville tro det tar 10 min fordi den laster ned java 19. Gjorde det for meg og etter at jeg oppdaterte pakker 🤔

Reply:
Laster den ned java 19? Hvorfor gjør den det? java 11 er spesifisert i `build.gradle` og den kjører på XP versjon 7.9.2. Jeg sliter med å forstå hvor java 19 kommer fra 😅 

Det er vel egentlig litt "ved siden av" denne tråden, men ved å kjøre `make install_new_dependencies` så vil man overskrive `package-lock.json` så avhengighetene en da vil bygge med vil ikke være lik de som ligger på github avhengig av når appen blir bygd. Noe å være klar over

Reply:
If running `make package` on fresh clones works fine on your systems as well, then I'd just keep the description as it was before. :)